pkg/deploy/deployment_che.go

```diff
@@ -130,6 +130,59 @@ func getSpecCheDeployment(deployContext *DeployContext, cmResourceVersion string
 		}
 	}
 
+	var cheEnv []corev1.EnvVar
+	cheEnv = append(cheEnv, selfSignedCertEnv)
+	cheEnv = append(cheEnv, gitSelfSignedCertEnv)
+	cheEnv = append(cheEnv, gitSelfSignedCertHostEnv)
+
+	identityProviderSecret := deployContext.CheCluster.Spec.Auth.IdentityProviderSecret
+	if len(identityProviderSecret) > 0 {
+		cheEnv = append(cheEnv, corev1.EnvVar{
+			Name: "CHE_KEYCLOAK_ADMIN__PASSWORD",
+			ValueFrom: &corev1.EnvVarSource{
+				SecretKeyRef: &corev1.SecretKeySelector{
+					Key: "password",
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: identityProviderSecret,
+					},
+				},
+			},
+		},
+			corev1.EnvVar{
```

Comment:
pls, format the lines

Reply:
Looks, like it well formatted file (with `gofmt`)